Sources/XcodeProj/Objects/Project/PBXProject.swift

```diff
@@ -21,7 +21,10 @@ public final class PBXProject: PBXObject {
     }
 
     /// A string representation of the XcodeCompatibilityVersion.
-    public var compatibilityVersion: String
+    public var compatibilityVersion: String?
+
+    /// An string representation of the PreferredProjectObjectVersion.
+    public var preferredProjectObjectVersion: String?
```

Comment:
Shouldn't this be an integer? do you have any example?

Reply:
Yes here: preferredProjectObjectVersion = 77;